core/src/main/scala/com/salesforce/op/stages/impl/classification/BinaryClassificationModelSelector.scala

```diff
@@ -231,9 +237,13 @@ case object BinaryClassificationModelSelector extends ModelSelectorFactory {
       trainRatio = trainRatio, seed = seed, validationMetric, stratify = stratify, parallelism = parallelism,
       maxWait = maxWait
     )
+    // For backwards compatibility, make sure evaluators always include the defaults
+    val allEvaluators = addDefaultEvaluators(trainTestEvaluators, Seq(
+      new OpBinaryClassificationEvaluator, new OpBinScoreEvaluator))
```

Comment:
these are still functionally hardcoded. what is the benefit of doing it this way?